plugins/prometheus/db_size.go

```diff
@@ -10,28 +10,28 @@ import (
 )
 
 var (
-	dataSizes *prometheus.GaugeVec
+	dbSizes *prometheus.GaugeVec
 )
 
-func registerDataMetrics() {
-	dataSizes = prometheus.NewGaugeVec(
+func registerDBMetrics() {
+	dbSizes = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "iota_data_sizes_bytes",
-			Help: "Data sizes in bytes.",
+			Name: "db_size_bytes",
+			Help: "DB size in bytes.",
 		},
 		[]string{"name"},
```

Comment:
The reason there is a label here is so that you can specify for what you want to know the size of via the label. That's why the metric name was generic. Therefore, either remove the label or revert back to the previous state.